src/main.rs

```diff
@@ -1,3 +1,46 @@
-fn main() {
-    println!("Hello, world!");
+use anyhow::Result;
+use solana_client::rpc_client::RpcClient;
+use solana_sdk::transaction::Transaction;
+
+
+// Settle the state on solana, called by sequencer
+pub fn settle_state(proof: String) -> Result<String> {
+    let rpc_client = RpcClient::new::<&str>("http://localhost:8899".into());
+
+    // TODO: Create transaction, still to be implemented
+    let transaction = Transaction::default();
+
+    println!("proof: {}", proof);
+
+    // Send transaction on chain - Currently will error with Invalid transaction. 
+    let settle_tx_hash = rpc_client
+        .send_and_confirm_transaction(&transaction)
+        ?;
+    Ok(settle_tx_hash.to_string())
+}
```

Comment:
Thanks for PR @imetandy. This looks like a good candidate for a separate file 👍🏽 Want to Add it to a separate module?

Will build on top of this tonight.